frontend/packages/shared/src/navigation/PackagesRouter/PackagesRouter.ts

```diff
@@ -0,0 +1,47 @@
+type ParamsOptions = {
+  org?: string;
+  app?: string;
+};
+
+type PackagesRoute =
+  | 'dashboard'
+  | 'editorOverview'
+  | 'editorUiEditor'
+  | 'preview'
+  | 'editorPublish';
+
+const packagesRoutes = {
```

Comment:
We should type this object to be a Record where the key matches the PackagesRoute. 😊 

```suggestion
const packagesRoutes: Record<PackagesRoute, string> = {
```